File: EPATEC/Models/Rol_Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EPATEC.Models
{
    public class Rol_Usuario
    {
        public long Id_rol_usuario { get; set; }
        public long Id_usuario { get; set; }
        public long Id_rol { get; set; }
    }
}

File: Mobile_App/java/models/RolUsuario.java
package models;

public class RolUsuario {
    public long id_Rol_Usuario;
    public long usuario;
    public long rol;

    public RolUsuario(){}

    public RolUsuario(long idRol, long idUsuario, long rol){
        this.id_Rol_Usuario = idRol;
        this.usuario = idUsuario;
        this.rol = rol;
    }
}
